Game/Scripts/PlayerJumpScript.cpp

```diff
@@ -0,0 +1,75 @@
+#include "PlayerJumpScript.h"
+
+#include "Application.h"
+#include "ModuleInput.h"
+
+#include "Components/ComponentTransform.h"
+#include "Components/ComponentRigidBody.h"
+
+REGISTERCLASS(PlayerJumpScript);
+
+PlayerJumpScript::PlayerJumpScript() : Script(), jumpParameter(2000.0f), canDoubleJump(false) , jumpReset(0)
+{
+	REGISTER_FIELD_WITH_ACCESSORS(JumpParameter, float);
```

Comment:
A suggestion from the scripting side, since you don't seem to be using the accessor methods for anything, you can use
```suggestion
	REGISTER_FIELD(jumpParameter, float);
```
And get rid of them entirely. This macro is intended to allow the game team to add additional logic when getting and setting a given field. For example, if one of the members of the class is computed from another, similar to what you're doing in `Start`.

Reply:
Fixed :)